reflex-algos/components/PySpark/hello_world/hello_world.py

```diff
@@ -0,0 +1,40 @@
+""" Hello World.
+Prints "Hello, world."
```

Comment:
May be say: Prints "Hello, world" by initializing spark context. This to demonstrate component can interact with Spark provided.